Dockerfile

```
@@ -11,7 +11,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION}
-RUN wget -O - -q https://raw.githubusercontent.com/client9/misspell/master/install-misspell.sh | sh -s -- -b /usr/local/bin/
+RUN wget -O - -q https://raw.githubusercontent.com/golangci/misspell/master/install-misspell.sh | sh -s -- -b /usr/local/bin/
```

```suggestion
RUN wget -O - -q https://raw.githubusercontent.com/golangci/misspell/master/install-misspell.sh v0.5.0 | sh -s -- -b /usr/local/bin/
```

https://github.com/golangci/misspell/blob/f33d7fdd6dd3f230fbd45f2c6295dc69895c1313/install-misspell.sh#L110
As shown above, if the architecture of this Docker image is `amd64`, the architecture name is converted to `64bit`.
However, with [the release of missspell v0.5.1](https://github.com/golangci/misspell/releases/tag/v0.5.1), assets are not prepared according to this rule.
Therefore, I think it is better to specify v0.5.0 for now.

---

A issue created: https://github.com/golangci/misspell/issues/20

---

https://github.com/golangci/misspell/issues/20 is resolved and CI successes.